scripts/createColumn.js

```diff
@@ -0,0 +1,19 @@
+const palette = require('./paletteUtils').palette;
+const Brick = require('./Brick');
+const TABLE_HEIGHT = require('../constants').TABLE_HEIGHT;
+
+
+// Creating a column by parent color
+module.exports = function(color) {
+    const column = document.createElement('div');
+    column.className = 'column';
+    const fragment = document.createDocumentFragment();
+    const currColor = palette(color);
+    for(i=0; i<TABLE_HEIGHT; i++) {
+      const COLOR = currColor();
```

Comment:
Color isn't constant, so shouldn't be in upper case, just suits name, like currentColor.
And rename this function, like getCurrentColor